### PR TITLE
feat: add Dockerfile

### PR DESCRIPTION
**Pull request by ekrem95**
Add Dockerfile based on `golang:1.12.3` as builder.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.12.3 as builder
+
+RUN go get github.com/drone/drone-yaml
+
+FROM golang:1.12.3
```
Comment (ferhatelmas) [inline Dockerfile]: use alpine here

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+COPY --from=builder /go/bin/* /usr/bin/
+
+WORKDIR /go
```
Comment (ferhatelmas) [inline Dockerfile]: not needed

Comment (ferhatelmas) [reply Dockerfile]: instead define binary as entry point

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12.3 as builder
```
Comment (ferhatelmas) [inline Dockerfile]: 1.12.4

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN go get github.com/drone/drone-yaml
+
+FROM alpine:3.9
+COPY --from=builder /go/bin/* /usr/bin/
+
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```
Comment (ferhatelmas) [inline Dockerfile]: why do we need this ?

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+ENTRYPOINT ["drone-yaml"]
+CMD ["--help"]
```
Comment (ferhatelmas) [inline Dockerfile]: line break at the end

File: Dockerfile (hunk 4 above)
Comment (ekrem95) [reply Dockerfile]: binary for `drone-yaml` couldn't be found
then i followed the example at https://stackoverflow.com/questions/34729748/installed-go-binary-not-found-in-path-on-alpine-linux-docker

Comment (ferhatelmas) [reply Dockerfile]: I see but better to disable cgo

`CGO_ENABLED=0` before go get

Comment (ferhatelmas) [reply Dockerfile]: since cgo is disabled, is this still needed ?

**Pull request merged**